OpenSim/Common/Component.h

```diff
@@ -1552,6 +1566,21 @@ template <class T> friend class ComponentMeasure;
     //the return type, @see addOutput()
     virtual void constructOutputs() {}
 
+    /// Invoke finalizeFromProperties() on the (sub)components of this Component.*/
+    void componentsFinalizeFromProperties() const;
+
+    /// Invoke connect() on the (sub)components of this Component.*/
+    void componentsConnect(Component& root) const;
+
+    /// Invoke addToSystem() on the (sub)components of this Component.*/
+    void componentsAddToSystem(SimTK::MultibodySystem& system) const;
+
+    /// Invoke initStateFromProperties() on (sub)components of this Component */
+    void componentsInitStateFromProperties(SimTK::State& state) const;
+
+    /// Invoke setPropertiesFromState() on (sub)components of this Component */
+    void componentsSetPropertiesFromState(const SimTK::State& state);
```

Comment:
All 5 of these methods mix doxygen comment styles. I presume `*/` will show up in the doxygen output.